### ISSUE TITLE
why need convert.py?

### ISSUE THREAD

**Issue by CSEEduanyu** [deepseek-ai/DeepSeek-V3#23]

**Comment by GeeeekExplorer** [comment]
`convert.py` is used to convert huggingface checkpoints into model parallel checkpoints required by the demo, so that each rank only needs to read the corresponding checkpoint when running the demo

**Comment by KKwanhee** [comment]
Why did `convert.py` exclude `layer61` (corresponding to `nextn_predict_layers`) from the generated `safetensor`? Doesn't that mean `layer61` won't be used during inference?

**Comment by GeeeekExplorer** [comment]
> Why was layer61 (corresponding to `nextn_predict_layers`) excluded in `convert.py` when generating the safetensor? Doesn't that mean `layer61` won't be used during inference?

Yes, The infer demo currently does not support MTP which uses layer 61.

**Comment by Huixxi** [comment]
But --model-parallel 32 seems also do not supported?

> > Why was layer61 (corresponding to `nextn_predict_layers`) excluded in `convert.py` when generating the safetensor? Doesn't that mean `layer61` won't be used during inference?
> 
> Yes, The infer demo currently does not support MTP which uses layer 61.
